src/tests/gui/Nav.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import Navbar from "../../Navbar";
+
+const renderNavbar = (scrollToExplore: () => void) => {
+  return render(<Navbar scrollToExplore={scrollToExplore} />);
+};
+
+describe("Navbar Component", () => {
```

Comment:
For visual unit tests, you can do snapshot testing as well. https://vitest.dev/guide/snapshot